index.js

```diff
@@ -0,0 +1,2 @@
+const userName = promp('Enter your name');
+alert(`Hello ${userName}!`);
```

Comment:
In this case it would seem better not to use string interpolation
```suggestion
alert('Hello ' + userName + '!');
```